docs/design/network-discovery/disable-port-discovery/domain-model.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+
+title Kilda Domain Model
+
+class Switch {
+   id : SwitchId
+
+   ...
+   __
+   unique constraint on switch_id
+}
+
+class PortProperties {
+   switch : Switch
+   port_no
+   discovery_enabled
+   __
+   unique constraint on switch_id+port_no
+}
+
+Switch --> PortProperties
```

Comment:
Shouldn't it be opposite direction?

Reply:
I assume relation "Switch -has/owns-> PortProperties" so decided to use this direction. We already have "Switch -has-> SwitchProperties" relation as well.